java/pom.xml

```diff
@@ -155,8 +168,12 @@
                 </executions>
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-                    <forkMode>once</forkMode>
+
+                    <!-- It's important to run tests serially because we are doing timing tests in some of the tests. -->
```

Comment:
It's necessary to run the tests serially for some reason.  Added this.  (might indicate a thread-safety issue in VW or our code).